fabric-dimensions-v1/src/testmod/resources/data/minecraft/dimension_type/fabric_dimension/void_type.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Example for fabric entity placer? also, does this need to go in a `minecraft/` subfolder like the dimension JSON?

Reply:
The entity placer is handled in code, see the example mod.

And yes this also needs to go in `minecraft/`

Reply:
ahhh, alright, sounds good